gulp/validateJson.ts

```diff
@@ -1,42 +1,42 @@
 import { promises as fs } from "fs";
-import Ajv, { ValidateFunction } from "ajv";
 import glob from "glob-promise";
 import log from "fancy-log";
-
-const ajv = new Ajv({ allErrors: true, strict: true });
-
-const baseSchemaPath = "./src/main/schemas";
-const langSchemaPath = `${baseSchemaPath}/lang.json`;
-const baseItemSchemaPath = `${baseSchemaPath}/item`;
-
-const validationConfigs: ValidationConfig[] = [
-  {
-    dataGlob: "./src/main/lang/*.json",
-    schemaPath: langSchemaPath
-  },
-  {
-    dataGlob: "./src/main/compendiums/item/ammo/**/*.json",
-    schemaPath: `${baseItemSchemaPath}/ammo.json`
-  },
-  {
-    dataGlob: "./src/main/compendiums/item/apparel/**/*.json",
-    schemaPath: `${baseItemSchemaPath}/apparel.json`
-  },
-  {
-    dataGlob: "./src/main/compendiums/item/magic/**/*.json",
-    schemaPath: `${baseItemSchemaPath}/magic.json`
-  },
-  {
-    dataGlob: "./src/main/compendiums/item/race/**/*.json",
-    schemaPath: `${baseItemSchemaPath}/race.json`
-  },
-  {
-    dataGlob: "./src/main/compendiums/item/weapon/**/*.json",
-    schemaPath: `${baseItemSchemaPath}/weapon.json`
-  }
-];
+import type { z, ZodError } from "zod";
```

Comment:
As the lint says.

Reply:
Oh, I didn't realise the commit hook allows linter warnings. I was assuming the linter was happy.

Reply:
It's not one of our own linters, I think that's just basic typescript. Github shows you warnings in a file. You would see the same when opening the file, but sadly there's no summary for all files, that I know of.